### PR TITLE
Fix valid redirect URIs for built-in account-console client on realm rename

### LINKED ISSUE #9541: Rename or import a realm is incomplete
### Describe the bug

Renaming a realm does not change "Valid Redirect URIs" and "Base URL" of client "account-console". The same problem occurs at import of a realm with new name. 

### Version

15.0.2

### Expected behavior

URIs and URL contains new realm part.

### Actual behavior

URIs and URL contains old string.

### How to Reproduce?

Rename a Realm and check account-console settings

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by jacekkow**
Fix valid redirect URIs for built-in account-console client on realm rename

Closes #9541

### CODE REVIEW

**Comment by jacekkow** [comment]
@ahus1: Second ExportImportManager was also updated.

As for tests, testing it properly requires:
- using `RealmManager` to create realm (so that automatically created clients are present),
- which requires properly set master realm,
- which can be created using `ApplianceBootstrap`,
- which requires provider of `org.keycloak.services.clientregistration.policy.ClientRegistrationPolicy`,
- which was not set up in my test env.

At that point I gave up.

**Comment by ahus1** [comment]
@jacekkow - Thank you for the change, I did a manual test and it works as expected.

I had a look and found an existing test where I added the rename check for the additional client. I ran the test locally and it succeeded. 

Once the tests pass, this is then ready to be merged.

**Comment by ahus1** [comment]
@jacekkow thank you for suggesting the additional test. I merged the content into the existing test as they were testing the same thing. Please give it a review if you think this is now ready to be merged.

**Comment by jacekkow** [comment]
@ahus1 - I went over it this morning and it looks good! Thank you!

**Comment by ahus1** [comment]
@sguilhen - can you please review this change? I already reviewed @jacekkow's changes, and I did a contribution about the updated thest. Thanks!

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/realm/RealmTest.java [hunk 1]
```diff
@@ -129,37 +130,65 @@ public void getRealms() {
...
+                        client.getClientId(),
+                        client.getRedirectUris()
+                                .stream()
+                                .map(redirectUri -> redirectUri.replace("/" + OLD + "/", "/" + NEW + "/"))
+                                .collect(Collectors.toList())
+                );
+            }
+        });
+
+        // at least those three default clients should be in the list of things to be tested
```
Comment (sguilhen) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/realm/RealmTest.java:165]: comment mentions three default clients but code only tests two?

**Review by sguilhen — COMMENTED** [review]
@ahus1 @jacekkow thanks for the PR! I have only one observation in the test, but otherwise it looks good to me!

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/realm/RealmTest.java (hunk 1 above)
Comment (ahus1) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/realm/RealmTest.java:165]: Hi @sguilhen - thank you for reviewing this. 

The tree default clients are `Constants.ADMIN_CONSOLE_CLIENT_ID, Constants.ACCOUNT_MANAGEMENT_CLIENT_ID, Constants.ACCOUNT_CONSOLE_CLIENT_ID`, and the code checks if they are part of the collections for `newBaseUrls` and `newRedirectUris` 

I hope this resolves your question.

Comment (sguilhen) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/realm/RealmTest.java:165]: oh right, I ended up looking only at the second line of each assert, my bad.

**Review by sguilhen — APPROVED** [review]

**Review by ahus1 — APPROVED** [review]
Approving as of @sguilhen's review

**Comment by ahus1** [comment]
Thank you @jacekkow for your efforts to fix this long-outstanding issue! It will be included in the next release of Keycloak.

**Pull request merged**